### DISCUSSION TITLE
Math in overbrace annotation

### DISCUSSION THREAD

**Discussion by cscherrer** [typst/typst#4062] [Q&A] [answered]
Hi,

This works fine:
```typst
overbrace(x(x + 1) ...(x + n - 1), "n factors")
```

But the "n" should really be math. If I do this
```typst
overbrace(x(x + 1) ...(x + n - 1), "$n$ factors")
```

The $s are printed in the content. I've tried lots of variations with no luck. What's the right way to do this?

**Comment by Coekjan** [comment, marked as answer]
This works fine (In math mode, you can use `#[ ... ]` to create a content block):

```typ
$ overbrace(x(x + 1) ...(x + n - 1), #[$n$ factors]) $
```

![image](https://github.com/typst/typst/assets/69834864/c3d185c6-7ab5-4384-b602-194287019d1b)

**Comment by cscherrer** [comment]
Perfect, thank you!!

**Comment by MDLC01** [comment]
Another possible solution is the following:
```typ
$ overbrace(x(x + 1) ... (x + n - 1), n "factors") $
```

**Reply by Coekjan** [reply]
This is more succinct!
